.github/workflows/verify.yml

```diff
@@ -1255,3 +1255,63 @@ jobs:
           debpkg/builddir/*.dsc
           debpkg/builddir/*.buildinfo
           debpkg/builddir/*.changes
+
+  packaging-conda:
+    needs: source-archive
+    if: github.event_name == 'pull_request'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+    defaults:
+      run:
+        shell: bash -el {0}
+
+    steps:
+    - name: Retrieve the xNVMe source archive
+      uses: actions/download-artifact@v4.1.1
+      with:
+        name: xnvme-src-archive
+
+    - name: Extract, and then remove, the xNVMe source archive
+      run: |
+        mkdir xnvme
+        mv xnvme-src.tar.gz xnvme/.
+        cd xnvme
+        tar xzf xnvme-src.tar.gz --strip 1
+        rm xnvme-src.tar.gz
+
+    - name: Set up Miniconda
+      uses: conda-incubator/setup-miniconda@v3
+      with:
+        auto-update-conda: true
+        miniconda-version: "latest"
+
+    - name: Setup env
+      run: |
+        conda env create -f xnvme/toolbox/conda/env.yaml
+        conda activate xnvme_conda_build
+        conda config --add channels conda-forge
+        conda config --set channel_priority strict
+        conda list
+
+    - name: Build Package
+      run: |-
+        cd xnvme/toolbox/conda
+        conda activate xnvme_conda_build
+        conda-build . --output-folder /tmp/conda_pkg
+        find /tmp/conda_pkg
+
+    - name: Find the built conda package
+      id: find-package
+      run: |
+        PACKAGE_PATH=$(find /tmp/conda_pkg -name "*.tar.bz2" | head -n 1)
+        echo "PACKAGE_PATH=$PACKAGE_PATH" >> $GITHUB_ENV
```

Comment:
Do we need this step?

Reply:
What do you mean?

Reply:
> Do we need this step?

Yes, to produce a path for "upload-artifact", making the built-artifact available.

Reply:
> Maybe I'm not reading the code correctly - but it seems to set an environment variable that we never use

hah, you are right, I had forgotten that I went with the entire build folder to get the auxiliary artifacts as well. So, nice catch!